---
output:
  pdf_document:
    fig_caption: yes
    keep_tex: yes
    latex_engine: pdflatex
    number_sections: no
    extra_dependencies: float
  word_document: default
  html_document:
    df_print: paged
header-includes:
- \usepackage{lineno}
- \linenumbers
- \usepackage{setspace}\doublespacing
- \usepackage{gensymb}
- \geometry{verbose,letterpaper,tmargin=2.54cm,bmargin=2.54cm,lmargin=2.54cm,rmargin=2.54cm}
#- \usepackage[nomarkers, nolists, figuresonly]{float}
bibliography: references.bib
csl: ecological-applications.csl
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
library(here)
```

## Quantifying Impacts of an Environmental Intervention Using Environmental DNA

Elizabeth Andruszkiewicz Allan$^{1*\dagger}$, Ryan P. Kelly$^{1*}$, Erin R. D'Agnese$^{1,3}$, Maya N. Garber-Yonts$^{1}$, Megan R. Shaffer$^{1}$, Zachary J. Gold$^{2}$, Andrew O. Shelton$^{2}$

$^{1}$ University of Washington, School of Marine and Environmental Affairs, 3737 Brooklyn Ave NE, Seattle, WA 98105, U.S.A.

$^2$ Conservation Biology Division, Northwest Fisheries Science Center, National Marine Fisheries Service, National Oceanic and Atmospheric Administration, 2725 Montlake Blvd. E, Seattle, WA 98112, U.S.A.

$^3$ Wild EcoHealth, Tacoma WA, 98465, USA

\vspace{1em}

$^{*}$ Authors contributed equally to this work.

$^{\dagger}$ Corresponding author: [eallan\@uw.edu](mailto:eallan@uw.edu){.email} \vspace{1em}

For submission to: \textit{Ecological Applications} \newline Manuscript type: Article \newline Open Research Statement: Data are already published and publicly available, with those publications properly cited in this submission. The repository for code can be found at: \newline  <https://github.com/eandrusz/quantitative_salmon_resubmit.git> \newline \textit{Keywords: environmental DNA, quantitative metabarcoding, environmental impact assessments, salmon, culvert}

\newpage

## Abstract

Environmental laws around the world require some version of an environmental impact assessment surrounding construction projects and other discrete instances of human development. Information requirements for these assessments vary by jurisdiction, but nearly all require an analysis of biological elements of ecosystems. Amplicon-sequencing - also called metabarcoding - of environmental DNA (eDNA) has made it possible to sample and amplify the genetic material of many species present in those environments, providing a tractable, powerful, and increasingly common way of doing environmental impact analysis for development projects. Here, we analyze a 18-month time-series of water samples taken before, during, and after two culvert removals in a salmonid-bearing freshwater stream. We also sampled multiple control streams to develop a robust background expectation against which to evaluate the impact of this discrete environmental intervention in the treatment stream. We generate calibrated, quantitative metabarcoding data from amplifying the 12s MiFish mtDNA locus and complementary species-specific quantitative PCR data to yield multi-species estimates of absolute eDNA concentrations across time, creeks, and sampling stations. We then use a linear mixed-effects model to reveal patterns of eDNA concentrations over time, and to estimate the effects of the culvert removal on salmonids in the treatment creek. We focus our analysis on four common salmonid species: cutthroat trout (*Oncorhynchus clarkii*), coho salmon (*O. kisutch*), rainbow trout (*O. mykiss*), and sockeye salmon (*O. nerka*). We find that one culvert in the treatment creek seemed to have no impact while the second culvert had a large impact on fish passage. The construction itself seemed to have only transient effects on salmonid species during the two construction events. In the context of billions of dollars of court-mandated road culvert replacements taking place in Washington State, USA, our results suggest that culvert replacement can be conducted with only minimal impact of construction to key species of management concern. Furthermore, eDNA methods can be an effective and efficient approach for monitoring hundreds of culverts to prioritize culverts that are required to be replaced. More broadly, we demonstrate a rigorous, quantitative method for environmental impact reporting using eDNA that is widely applicable in environments worldwide.

\newpage

## Introduction

At present, it remains difficult to comprehensively measure the environmental impacts of discrete human activities, despite such assessment often being required by law. Within the United States, both state and federal laws often require a form of environmental-impact assessment for medium- to large-scale projects (i.e., those that might have a significant impact on the environment) [@morgan2012]. Outside the US, many nations have their own versions of these same laws. Specifically when measuring impacts on aquatic ecosystems, assessments generally are based on literature reviews or field measurements of key species selected beforehand [@rubin2017]. These traditional methods are often expensive, rely on just a few species, and are limited in spatial and temporal coverage [@martin2012]. Moreover, they often lack pre-project monitoring and any or sufficient post-project monitoring, given that the goals of a development project normally focus on construction itself and funding is often extremely limited. For example, a recent literature review of stream restoration projects cited that more than half of projects evaluated (62%) had no pre-project monitoring and only sampled once per year (for before, during, and post-project sampling) [@rubin2017]. Thus, current assessment efforts relying on traditional survey methods often fall short in documenting and quantifying environmental impacts.

A key difficulty in conducting ecosystem assessments is that there is no one way to survey the world and just "see what is there." All methods of environmental sampling are biased as they capture a selective portion of the biodiversity present [@rubin2017]. Net samples for fish, for example, fail to capture species too small or too large to be caught in the net. Environmental DNA (eDNA), however, comes as close to this goal as any method yet developed although not without bias (see below): a sample of water, soil, or even air, contains the genetic traces of many thousands of species, from microbes to whales. Sequencing eDNA is therefore a means of surveying many species in a consistent and scalable way [@taberlet2012; @thomsen2015]. Environmental assessments have begun to make use of eDNA for such work around the world [@moss2022; @maasri2022; @klein2022; @duda2021; @muha2017], but are not yet common practice. Surveying the world by eDNA has long been commonplace in microbial ecology [@ogram1987; @rondon2000; @turnbaugh2007] but has recently become popular for characterizing eukaryotic communities [@taberlet2012; @kelly2014; @port2015; @devargas2015; @valentini2016; @stat2017]. Techniques generally include an amplification step such as quantitative PCR, digital or digital-droplet PCR, or traditional PCR from mixed templates followed by high-throughput sequencing [@ruppert2019]. This last technique is known as eDNA metabarcoding.

In a metabarcoding approach, broad-spectrum PCR primers identify hundreds or thousands of taxa across a very wide diversity of the tree of the life (e.g., @leray2013). Nevertheless the absence of a taxon from a sequenced sample does not indicate the absence of that taxon from the environment but rather that the taxon failed to amplify [@gold2023; @shelton2016; @kelly2019; @buxton2021]. In virtually all comparisons, metabarcoding recovers far more taxa than any other sampling method [@port2015; @kelly2017; @seymour2021]. However, we expect results from metabarcoding to differ dramatically from non-PCR based sampling methods due to the fundamental differences in sampling residual genetic material as opposed to whole organisms. Furthermore, eDNA analyses rely on several laboratory processes, including PCR amplification, all of which contribute to complicating the interpretation of results (see @shelton2016 and @kelly2019). Specifically, PCR amplification is an exponential process for which the efficiency varies across species and primer set [@gloor2016]. By understanding these differences, we can correct for taxon-specific biases to yield quantitative estimates of the community composition prior to PCR [@mclaren2019; @shelton]. Other ways to conduct quantitative metabarcoding include using qSeq [@hoshino2021], a process in which a random tag is added to target sequences before PCR. However, if different species amplify at different rates during PCR, these quantifications would reflect not just the starting concentration but also the amplification efficiency.

After correcting for amplification biases, the resulting dataset is compositional, revealing the proportions of each species' DNA present in each sample, but importantly, contains no information about the absolute abundance of DNA present [@gloor2016; @mclaren2019; @silverman2021; @shelton]. We can tie these proportional estimates to absolute abundances using additional data such as a quantitative PCR (qPCR) assay for one of the taxa present. Thus, a single qPCR assay and a single metabarcoding assay can together provide quantitative estimates of many species as opposed to running as many qPCR assays as species of interest (see also [@pont2022]). Together, we can use these data to assess changes in eDNA concentrations of species over time, and due to environmental impacts, such as replacing a culvert under a road.

As a result of a ruling in a federal court [@martinez2013], Washington State is under a mandate to replace hundreds of culverts that allow water to pass under roads and highways, costing billions of dollars. Improperly designed culverts can lead to many negative consequences for fish, especially anadromous salmon, including habitat fragmentation, loss of accessibility to spawning and rearing habitat, and genetic isolation [@price2010; @frankiewicz2021]. These impacts on salmonids furthermore violate the sovereign treaty rights of the region's indigenous tribes [@martinez2013]. Salmonid species are of cultural and economic importance to the indigenous peoples of the region, and without restoration of historic salmon-rearing habitat, the continued decline of salmonids can lead to not only ecological destruction, but the loss of cultural and economic viability for many indigenous tribes [@schmidhauser1976a; @lackey2003; @long2018].

Presently, the prioritization process for replacing culverts preventing fish passage conducted by the Washington Department of Transportation is a protocol provided by the Washington Department of Fish and Wildlife, which includes factors such as the amount of habitat blocked by the barrier, the types of species blocked by the barrier, and estimated cost of repair, among other things [@washingtondepartmentoffishandwildlife2019]. However, data on fish presence upstream of barriers (i.e., culverts blocking fish passage) are rare and often not included in these assessments. Using eDNA as a proxy for fish presence could provide important data for project prioritization and have the potential to be more cost effective.

Once a culvert has been designated as in need of repair, the intention is to improve conditions for biota, including migrating fish, but the construction itself might have a short-term negative effect before the longer-term improvements are realized. Specifically in culvert replacements, studies have cited the negative impacts of construction to include sediment accumulation, removal of vegetation, and blocking flow and stranding fish [@washingtondepartmentoffishandwildlife2019; @wellman2000]. However, it is unclear how long these effects might last and if the long-term benefits of the culvert replacement justify the short-term costs of the construction. These disruptions also underscore the importance of both properly assessing culverts to determine if they are blocking fish passage and monitoring after construction to ensure the replacement actually improved fish passage.

Many studies have attempted to quantify when culverts are barriers to fish passage and how effective culvert replacements are for fish passage, either by measuring physical parameters of the culvert and stream after replacement [@price2010], or by measuring biological parameters, including electrofishing [@ogren2015] or genetic differentiation from fish tissues [@nathan2018; @wood2018]. In some cases, culverts deemed blockages did not prove to block fish passage [@macpherson2012], while in others, blockages that were replaced were not found to improve fish passage [@price2010] or improve overal biotic integrity [@ogren2015]. Sampling water for eDNA analysis before, during, and post-restoration can provide valuable information on if the restoration is needed, how the restoration negatively impacts communities during construction, and if the restoration efforts did in fact correct the blockage.

Here, we report the results of an approximately 18-month eDNA sampling effort before, during, and after the replacement of two culverts (one small and one large) in a creek, assessing the impact of these projects on the salmonid species present. We do so using a combination of metabarcoding (12s mtDNA) and qPCR to yield estimates of the concentrations of DNA present at each time point, and we use parallel samples from four control creeks to develop a causal analysis of changes in these concentrations. A clear opportunity for policy-relevant eDNA work is in using its power to survey many species at a time to improve the way we assess the impacts of human activities. Here, we demonstrate the utility of eDNA for policy-relevant environmental assessments by surveying many species simultaneously and improving the way we assess the impacts of human activities.

## Methods

### Site and Species Selection

We selected sampling locations based loosely on an asymmetrical BACI (Before-After-Control-Impact) study design [@underwood1992; @underwood1994; @benedetti-cecchi2001] to measure the environmental impact of a culvert replacement using eDNA. We sampled four control creeks where construction was not occuring, (Figure 1) at monthly intervals, both upstream and downstream of each creek's culvert. The two culverts in the treatment creek (Padden) were suspected to be partially impassible and thus were removed and replaced during the course of the study. The four control creeks ranged from preventing fish passage (Barnes and Chuckanut), partially passable (Squalicum), to allowing fish passage (Portage; see Appendix S1) [@washingtondepartmentoffishandwildlife2019]. These creeks were chosen due to their comparable size, flow, watersheds, and species presumed to be present to constrain as many ecological variables as possible.

The first culvert replacement (SR-11) in Padden Creek occurred over about two months and included the "de-watering" of the creek, removal of the existing culvert, installation of the new culvert, and then the "re-watering" of the creek from late August 2021 to early October 2021 (Appendix S1: Figure S4). The second culvert replacement (I-5) in Padden Creek was a much larger construction project, including daylighting the creek and building a bridge under a large, five-lane interstate. In-water work for the I-5 culvert replacement began in late June 2022 and was completed in September 2022. By sampling before, during, and after both construction events, we were then able to isolate the effect of the culvert replacement itself -- controlling for temporal trends, background environmental variability, and sampling variability -- using a linear mixed effects model of eDNA abundances across creeks, time points, sampling stations, and species.

Because salmonids are the primary species of management concern in these creeks, we focus the present analysis on the four salmonid species most common in our data: cutthroat trout (*Oncorhynchus clarkii)*, coho salmon (*O. kisutch*), rainbow/steelhead trout (*O. mykiss*), and sockeye/kokanee salmon (*O. nerka*). Not all four salmonids are expected to be found in all five of the creeks sampled. As documented by WA Department of Fish and Wildlife SalmonScape (<http://apps.wdfw.wa.gov/salmonscape/map.html>), all creeks contain cutthroat trout, steelhead trout, and coho salmon. Barnes Creek is the only creek documented to have kokanee salmon (a freshwater sub-type of sockeye salmon). However, local spawner surveys conducted by the City of Bellingham from 2015-2020 in Padden Creek documented kokanee salmon, as well as the other three species [@cityofbellingham2015]. The four salmonid species in this study have different life histories and behaviors that would impact when fish (and therefore eDNA concentrations) occur in the creeks. Furthermore, three of the four species in this study have both freshwater resident and saltwater migrating behavior. For the fish exhibiting migratory behavior, the run timings vary for each species in the study area (see Discussion and Appendix S1: Figure S4). Therefore, our eDNA concentrations might reflect contributions from both migrating and non-migrating individuals at any given time point in the dataset.

### Water Sampling

From March 2021 to February 2022, all five creeks were sampled monthly (n=12). Monthly sampling continued in Portage Creek, Padden Creek, and Squalicum Creek through August 2022, with one additional sampling point in December 2022 (n=19). At each sampling station (upstream and downstream of a culvert) at each creek, we collected three 2-liter water samples. Samples were collected using an eDNA Backpack (Smith Root; @thomas2018), a portable pumping-and-filtering device set to filter at 1 L/min at 82.7 kPa (12 psi). In some months, less than 2 L of water was filtered due to clogging. Water samples were filtered using single-use inlet tubes through 5$\mu$m self-preserving filters (Smith Root, Vancouver, WA), which were then dried and kept at room temperature until DNA extraction within 1 month of collection [@thomas2019].

Over the course of the sampling, water discharge varied from very low to no flow in summer months to high flow in winter months (Figure 2). We account for this dilution by converting eDNA concentration [copies/$\mu$L] to an eDNA mass flow rate [copies/s] by multiplying eDNA concentrations by discharge [L/s] [@thalinger2019; @tillotson2018]. Flow gauges maintained by the United States Geological Survey (USGS) were used for Padden Creek (USGS Gauge 12201905), Chuckanut Creek (USGS Gauge 12201700), and Squalicum Creek (USGS Gauge 12204010; <https://maps.waterdata.usgs.gov/mapper/index.html>; @u.s.geologicalsurvey1994; Appendix S1: Figure S1). Over the course of sampling, the flow gauges at Chuckanut Creek and Squalicum Creek became inoperable after a major flooding event. To find discharge rates for Chuckanut and Squalicum Creeks, five years of historical data (2015-2020) were used to generate a monthly averaged correction factor based on Padden Creek (Appendix S1, Appendix S1: Figure S3). No discharge data was available for Portage Creek or Barnes Creek. Based on field sampling conditions, the discharge from Padden Creek was used as a proxy for both Portage and Barnes as they are in similarly sized watershed areas and land-cover characteristics.

### DNA Extraction, Amplification, Sequencing

All molecular work prior to sequencing was performed at the University of Washington. Details of the molecular work can be found in Appendix S1. Briefly, DNA was extracted off filters using a Qiashredder column (Qiagen, USA) and the DNeasy Blood and Tissue Kit (Qiagen, USA) with an overnight incubation (Appendix S1, @thomas2019). Extracts were stored at -20\degree C until PCR amplification within 2 months of extraction.

For metabarcoding, we targeted a \~186 bp hypervariable region of the mitochondrial DNA 12S rRNA gene for PCR amplification (MiFish; Miya et al. 2015), but using modified primer sequences as given in Praebel and Wangensteen (unpublished; via personal communication). The primer sequences, final reaction recipe, and cycling conditions can be found in Appendix S1. Each month of samples was amplified on a single plate with the addition of a no template control (NTC; molecular grade water in lieu of template) and a positive control (genomic DNA from kangaroo, a species not present in the environment). PCR products were visualized, size-selected, and diluted iteratively if inhibited. After cleaning, a second PCR amplification added unique indices to each sample using Nextera indices (Illumina, USA) to allow pooling multiple samples onto the same sequencing run (See Appendix S1 for details). Indexed PCR products were also size-selected and visualized before pooling for sequencing. Samples were randomized in 3-month blocks and each block split across 3 sequencing runs to avoid run effects, for a total of 14 sequencing runs. The loading concentration of each library was 4-8 pM and 5-20% PhiX was included depending on the composition of the run. Sequencing was conducted using an Illumina Miseq with v3 2x300 chemistry at the NOAA Northwest Fisheries Science Center and the University of Washington's Northwest Genomics Center.

Here, we used mock communities to determine the species-specific amplification efficiencies for each salmonid in the study. We constructed five communities with known proportions of starting DNA from different species (total DNA as measured by Qubit). The communities ranged from having a total of 12 to 20 species, but six salmonid species were included in all five mock communities (Appendix S3: Table S2). We sequenced these communities using the same metabarcoding primers and thermocycling conditions above and then determined the species-specific amplification rates given the discrepancy between the known starting proportion and the proportion of reads after sequencing. The mock community data were then used to correct the sequencing reads from the environmental samples to estimate the starting DNA proportions of each species in environmental samples, which is the metric of interest (Figure 3, green boxes). This is the first application of the model to correct eDNA data from water samples with mock community data as described in @shelton (see Appendix S2 for more information).

### Bioinformatics

After sequencing, bioinformatic analyses were conducted in R [@rcoreteam2017]. A more detailed description of the bioinformatics pipeline is included in Appendix S1. Briefly, primer sequences were removed using *Cutadapt* (Version 1.18) [@martin2011] before *dada2* [@callahan2016] trimmed, filtered, merged paired end reads, and generated amplicon sequence variants (ASVs). Taxonomic assignment was conducted via the *insect* package [@wilkinson2018] using a tree generated by the developers for the MiFish primers that was last updated in November 2018. Only species level assignments from *insect* were retained and ASVs not annotated or not annotated to species level were then checked against the NCBI nucleotide database using BLAST+ [@camacho2009]. Query sequences that matched a single species at \>95% identity were retained.

### Quantitative PCR and Inhibition Testing

We quantified cutthroat trout (*O. clarkii*) DNA in each sample, targeting a 114 bp fragment of the cytochrome b gene with a qPCR assay [@duda2021]. The primer/probe sequences, final recipe, and thermocycling conditions can be found in Appendix S1. Each DNA sample was run in triplicate and was checked for inhibition using the EXO-IPC assay (Applied Biosystems). The majority of environmental samples (60%) were inhibited and accordingly diluted for analysis. In 80% of inhibited samples, a 1:10 dilution or less remedied the inhibition, but some samples (11%) required dilution by a factor of up to 1000. Each plate included a 8-point standard curve created using synthetic DNA (gBlocks) ranging from 1 to 100,000 copies/$\mu$L and six no template controls (NTCs) were included on each plate with molecular grade water instead of template. All qPCRs were conducted on an Applied Biosystems StepOnePlus thermocycler.

All qPCR data was processed in R using Stan [@standevelopmentteam2022], relating environmental samples to the standard curve via a linear model (Figure 3, blue boxes; Appendix S2: Figure S13). We amended the standard linear regression model to more realistically capture the behavior of qPCR observations, accommodating non-detections as a function of underlying DNA concentration, and letting the standard deviation vary with the mean (lower-concentration samples had more uncertainty). See @mccall2014 and @shelton2019 for similar models; see Appendix S2 for full statistical details. Subsequent analysis corrected for sample-specific dilution if found inhibited and corrected for any variation in water-volume filtered during sample collection. Samples with standard deviations between technical replicates larger than 1.5 Ct values were removed from analyses.

### Quantitative Metabarcoding

The intercalibration of the mock community samples demonstrated the rank order of amplification efficiencies for salmonids (Appendix S1: Figure S10 and Appendix S1: Figure S11). Cutthroat trout (*O. clarkii*) and sockeye/kokanee salmon (*O. nerka*) had similar amplification efficiencies, both of which were higher than rainbow/steelhead trout (*O. mykiss*) and coho salmon (*O. kisutch*), which had the lowest amplification efficiency. Calibrated metabarcoding analysis yielded quantitative estimates of the proportions of species' DNA in environmental samples prior to PCR. We then converted these proportions into absolute abundances by expansion, using the qPCR results for our reference species, cutthroat trout (*O. clarkii*). We estimated the total amplifiable salmonid DNA in environmental sample $i$ as $\text{C}_{\text{amplifiable}_{i}} = \frac{\text{C}_{\text{qPCR reference}_{i}}}{\text{Proportion}_{\text{reference}_{i}}}$, where C has units of [DNA copies/uL] and then expanded species' proportions into absolute concentrations by multiplying these sample-specific total concentrations by individual species' proportions, such that for species $j$ in sample $i$, $\text{C}_{i,j} = \text{C}_{\text{amplifiable}_{i}} * \text{Proportion}_{i,j}$. Here, we combine the modeled output of the qPCR model for cutthroat trout (Figure 3, dashed blue box) and modeled proportions of salmonid DNA from metabarcoding (Figure 3, dashed green box). Although in the future this could be used as a joint model, here the precision of our modeled estimates were very high such that we used the mean of the posterior estimates from each model to move forward as input to the time series model (Figure 3, dashed purple box; see Appendix S2 for more details). Finally, due to the range of water discharge over the course of the year, we converted from DNA concentration [copies/L] to a mass flow rate [copies/s] after multiplying by the discharge of each creek [m\textsuperscript{3}/s] (Figure 3, solid purple boxes).

### Estimating the Effects of Culvert Replacement and of Culverts Themselves

We sampled four control creeks as context against which to compare the observations in Padden Creek, our treatment creek where the two culverts were being replaced. At a given station in a given creek, some DNA concentration exists for each species. For simplicity, we focus on a single species and a single station (downstream or upstream) for the moment. Our observations of the (log) DNA concentration in creek $i$ at time $t$ are distributed as $Y_{i,t} \sim \mathcal{N}(\mu_{i,t},\,\sigma^{2})$. More complex versions of the model may let $\sigma$ vary across creeks, time points, species, or with environmental covariates of interest.

We are interested in how the DNA concentration changes over time, so we assert that the expected value of DNA in a creek at time $t$, $\mu_{i,t}$, depends upon its time point, in some way. We considered three ways of modeling the salmonid eDNA data, each in a Bayesian framework, but each treating non-independence among time points somewhat differently:

-   A linear auto-regressive (AR(1)) model, written in `stan`. For each species in each creek, the expected concentration of eDNA of each month is a linear function of the expected value from the previous month. Within a species, the monthly autoregressive parameters are shared across creeks.

-   A generalized additive model (GAM), written in `brms` (which itself writes a `stan` model). For each species in each creek, an independent set of spline (weighting) parameters describes the temporal trends in expected eDNA concentration; the number of spline knots is shared across species and creeks.

-   A linear mixed-effects (LME) model, written in `rstanarm`. For each species in each creek, sampling month is treated as a random effect. Each species-creek-month effect is treated as an independent draw from a common distribution. 

Ultimately, the three models yielded very similar results (Appendix S2), and the LME model proved simplest and most flexible insofar as it could easily handle datasets with uneven sets of observations -- for example, cases in which a species was detected downstream of a barrier, but not upstream.

In `R` code using `rstanarm`, this model is coded as

    stan_glmer(log(observed) ~ (1 + time_idx|creek:species) + (1|station:creek:species:time_idx)  

## Results

### Metabarcoding and Quantitative PCR

In total, we generated \~52 million reads across all environmental samples and 27 mock community samples (3 communities x 9 replicates [6 even, 3 skewed proportions]) for calibration (see below). After quality-filtering and merging all runs, \~45 million reads remained from \~24,000 amplicon sequence variants (ASVs) in the environmental samples, of which \~83% of reads were annotated to species level (per sample: mean = 82%, median = 93%, min = 0%, max = 99.99% of reads annotated). We only focus on the metabarcoding data from four salmonids for the remainder of this paper. The four salmonids represent \~54% of all reads and \~64% of the annotated reads found in environmental samples.

In the mock community samples, 98.7% of the \~5 million reads after quality filtering were annotated to species level. Importantly, the target salmonid ASVs in the mock communities were found in environmental samples, unambiguously linking the taxa in calibration samples with those in environmental samples. The most common salmonid species found in the environmental samples was cutthroat trout (*O. clarkii*), which was found in \~85% of samples, followed by coho salmon (*O. kisutch*) found in \~62% of samples, then rainbow/steelhead trout (*O. mykiss*) found in \~40% of samples, and finally sockeye/kokanee salmon (*O. nerka*) found in \~10% of samples. Over half of the samples across all times, creeks, and stations had at least 50% of reads assigned to cutthroat trout.

After calibrating metabarcoding data using mock communities (See Appendix S1 and Appendix S2), we estimated the salmonid composition across time points, creeks, and stations (Figures 4 and 5). The culvert in one control creek (Barnes) appeared to be nearly a total barrier to salmonid passage, with salmonid eDNA detected upstream of the culvert at only three time points, in contrast to being detected at every time point in the downstream station of the same creek. The other four creeks had no such pattern associated with the culverts, suggesting that fish passage may have been possible through the culverts, or that there were resident populations upstream of the culverts.

All environmental samples were quantified for absolute concentrations of cutthroat trout DNA across 32 qPCR plates, resulting in \~630 samples (\~60%) with a positive detection in at least 1 of 3 technical replicates. The modeled output of cutthroat trout DNA concentrations, ranged from 50 copies/L to 1.4 x 10\textsuperscript{6} copies/L, with a mean value of \~47,000 copies/L (Figure 6).

We combined compositional information from metabarcoding with absolute concentrations from qPCR for our reference species, cutthroat trout (*O. clarkii*), to estimate the total concentration of DNA for each species (See Appendix S2). These quantitative data for all four target species were then used in the linear mixed-effects model to assess salmonid trends over time (Figure 7).

### Effects of Culverts

Before considering the effect of construction, the difference in trends between upstream and downstream stations (Figure 7) demonstrates that the culverts themselves have some effect, but generally not a large effect on the salmonid species surveyed. A notable exception was Barnes Creek, as the culvert was so clearly a barrier as most time points had no salmonid DNA upstream. Padden Creek upstream of I-5 also was more clearly a barrier to fish passage, while the culvert across SR-11 seemed to be less of a barrier to fish passage. In other cases, salmonid DNA is found upstream but not downstream, indicating that the culvert is likely not a barrier and there are resident individuals upstream of the culvert.

Summarizing over all species and the four creeks used in the time series model, the culvert effect was minimal (Figure 8); the average log-fold change between upstream and downstream sites was not significantly different from zero. Individual species' patterns were similar, indicating that there is not a species-specific effect where culverts block the passage of some salmon but not others (Appendix S1: Figure S12). The maximum positive log-fold change (i.e., upstream having a higher mass flow rate) was 2.78 in Squalicum Creek for coho salmon (*O. kisutch*) in August 2021, while the maximum negative log-fold change (i.e., downstream having a higher mass flow rate) was -1.11 found in Squalicum Creek for cutthroat trout (*O. clarkii*) in December 2021 (Appendix S1: Figure S12). Of all species, creeks, and time points, 23 of the 151 observations were within a log-fold change of -0.1 to 0.1, which corresponds with eDNA mass flow rates upstream within 10% of mass flow rates downstream.

We also considered how the log-fold change in eDNA mass flow rates were impacted by the flow rate or discharge of the creek itself (Figure 9). We found that at months of the lowest flow (summer months), the log-fold changes between mass flow rates were the highest, while in winter months with highest discharge the log-fold changes were lower and downstream sites often had higher eDNA mass flow rates than upstream sites (Appendix S1: Figure S12).

### Effects of Culvert Replacement

By comparing the difference in upstream and downstream mass flow rates before and after construction in Padden Creek, we can assess how large of an impact the two culvert replacements had on salmonid species (Figure 10). The effects of the culvert replacement operations appeared to have been transient and fairly minor for the four salmonid species surveyed. We saw very minor fluctuations in the difference between upstream and downstream salmonid DNA mass flow rates, and did not see an increase in this difference due to the culvert removal as the log-fold changes in Padden Creek were similar to those in the control creeks at the same time points (Figure 10, grey points vs. black points in areas of yellow shading).

## Discussion

### Environmental DNA can provide quantitative measurements of environmental impacts

Here, we used both eDNA metabarcoding and a single species-specific qPCR assay to rigorously quantify both the effect of culverts and the impact of two culvert replacements on salmonids in the same creek. We observed a clear seasonal pattern in the DNA concentrations of four salmonid species detected in the study. The sampling design and the linear mixed effects model leveraged information across treatment and control creeks to integrate the change in eDNA concentrations due to time, whether a sample was collected below or above a barrier (i.e., culvert), and whether or not there was construction occurring. Thus, we could isolate the changes in eDNA concentrations as a result of the intervention (i.e., construction) while accounting for the variance due to time and station (i.e., season and culvert).

A few other studies have used eDNA to measure environmental impacts in rivers and streams. @duda2021 used 11 species-specific qPCR assays to document the distribution of resident and migratory fish after a large dam removal project (Elwha River near Port Angeles, Washington). No eDNA sampling was conducted before the dam removal, but the study provided a wealth of information about species returning after the dam removal, providing a very important dataset to use eDNA to monitor ecological changes due to human intervention. Similarly, @muha2017 sampled three locations upstream and three locations downstream before and after the removal of a weir that was thought to be a barrier to salmonid migrations. The authors only sampled once before and twice after the removal, spanning about a year, and used eDNA metabarcoding to look at the presence/absence of species detected. They found that in fact the before sample demonstrated that the weir was not preventing fish passage (similar to the results found in this study) and furthermore documented a slight increase in alpha diversity in the first time point after the barrier removal and then a return to a similar alpha diversity in the second time point after the removal (similar results found in this study using eDNA concentrations rather than diversity). Finally @yamanaka2016 sampled along a river with three barriers, finding some fish able to cross barriers and some not, suggesting that the eDNA can indicated habitat connectivity for fishes across barriers.

Importantly, our study demonstrates the value of combining a single qPCR assay with metabarcoding data to generate quantitative estimates of eDNA concentrations of many species without requiring *n* qPCR assays for *n* species of interest. Here, we ultimately only quantified the impacts of four species, but importantly, we did not know *a priori* how many species of interest there might be and we reduced our efforts two fold by only conducting two assays (one species-specific qPCR and one metabarcoding assay) as opposed to four assays (four species-specific qPCR assays). This can also be particularly helpful for taxa that don't have a previously published qPCR assay, but are detected using universal metabarcoding assays. Metabarcoding data alone only gives compositional data, which cannot be used in a time series to quantify environmental impacts because there is no information about absolute eDNA concentrations. However, by anchoring or grounding proportions using a single qPCR assay, the proportional data can be turned into quantitative data. The species for which to run the qPCR assay can be determined after the metabarcoding is completed; the most commonly found species with a robust qPCR assay should be used to glean the most information.

### Fish life histories and expected patterns

The four salmonid species in this study have different life histories and behaviors that would impact when fish (and therefore eDNA concentrations) occur in the creeks. Three of the four species in this study have both freshwater and anadromous populations. Cutthroat trout (*O. clarkii*) encompasses both non-migrating, resident trout in the creeks and coastal run cutthroat that migrate into Padden Creek from saltwater (Bellingham Bay). Similarly, *O. nerka* includes both anadromous sockeye salmon and freshwater resident kokanee salmon and *O. mykiss* includes both anadromous steelhead trout and non-migrating rainbow trout. Using eDNA, we cannot distinguish between the migrating and non-migrating subspecies of *O. clarkii*, *O. nerka*, and *O. mykiss*. Therefore, our eDNA concentrations might reflect contributions from both migrating and non-migrating individuals at any given time point in the dataset. 

For these four anadromous salmonids, the run timings for the migrating populations vary for each species in the study area (Bellingham, WA). Adult coastal cutthroat (*O. clarkii*) are documented to run throughout the entire year, whereas coho salmon (*O. kisutch*) run from September to December, sockeye salmon (*O. nerka*) run from October to December, and steelhead trout (*O. mykiss*) run from November to June. For migrating coho (*O. kisutch*) and steelhead trout (*O. mykiss*), juveniles may be present in the creeks year-round (Appendix S1: Figure S4). eDNA methods at present cannot distinguish adults versus juveniles from DNA found in a water sample.

Despite the mix of migrating and non-migrating populations and various run timings, our metabarcoding data demonstrate that in Padden Creek, there was a clear signal of sockeye/kokanee salmon (*O. nerka*) both upstream and downstream only in November 2021 - February 2022 and again in December 2022. This signal corresponds well with the documented run timing of October to December and the presence of out-migrating juveniles in early spring. In contrast, cutthroat trout (*O. clarkii*) and coho salmon (*O. kisutch*) were found nearly year-round in Padden Creek. The persistent signal from *O. clarkii* could be explained by resident cutthroat trout. However, *O. kisutch* does not have a resident subspecies and the run timing is only documented from September to December. This could potentially be due to juveniles maturing and residing in the creeks for 1-2 years after hatching while adults migrate into the creeks only during the run time to spawn. Visual surveys (e.g., snorkel surveys, electrofishing, smolt traps) are conducted infrequently to determine adult and juvenile salmonid abundances. Though *O. kisutch* eDNA was found year round, the highest concentrations were found near the expected run timing and the life history of *O. kisutch* includes rearing year-round in freshwater. Finally, though the lowest concentrations on average, rainbow/steelhead trout (*O. mykiss*) was also found nearly year-round in Padden Creek, which could be contributions from migrating steelhead (November to June), juveniles maturing and migrating, or from resident rainbow trout. Though the *O. mykiss* signal is found year-round, the highest concentrations do seem to correspond with the steelhead run timing.

### Interpreting eDNA with respect to fish abundance and flow

By capturing residual eDNA from water samples, we are measuring a different signal than counting how many fish are in the creek at each time of sampling. We should not expect the eDNA concentration to directly correlate to the number of fish in the creek at the time of sampling. @shelton2019 used a paired eDNA sampling and seine netting analysis to demonstrate that eDNA concentrations provide a smoothed biological signal over space and time. We acknowledge this smoothing effect and emphasize that in the context of using eDNA for environmental impact assessments, it is preferable to use a survey technique such as eDNA that integrates signal across a larger spatial and temporal scale.

Many previous papers have commented on the "ecology" of eDNA and the various processes that contribute to eDNA concentrations in environmental samples (e.g., shedding rates, decay rates, transport) [@barnes2015]. For example, higher concentrations of eDNA could be the result of a greater number (or biomass) of fish present, or increased shedding rates, or decreased decay. Many review papers document the nuances of interpreting eDNA data and we recommend reviewing them for a deeper understanding (see @andruszkiewiczallan2020 for a review on shedding and decay rates and @harrison2019 for a review on transport). Other studies have also documented the relative importance of eDNA transport in streams. Most notably, @tillotson2018 measured eDNA at four sites with similar discharge rates to the creeks in this study and specifically addressed spatial and temporal resolutions, finding that eDNA concentrations reflect short time- (and therefore length-) scales by comparing peaks in eDNA concentrations to counts of salmon and accumulation by measuring both upstream and downstream sites. The authors found that the sampling site furthest downstream did not accumulate eDNA and that two tributaries feeding into a main channel were additive [@tillotson2018]. For more general models and empirical data documenting transport distances in streams, see @wilcox2016, @jane2014, @jerde2016, @shogren2016, and @civade2016. Certainly eDNA concentrations can arise from different scenarios and future work should continue to investigate how to tease apart the nuances of relating eDNA concentrations to fish abundance.

In this study, to assess the impact of a culvert on fish passage, we compare eDNA concentrations upstream and downstream at the same time point in a given creek. The distance between the upstream and downstream sampling was minimal (\~60-300 m, average distance of \~150 m). Therefore, we assume that the small differences in spatial and temporal scale between sampling locations is minimal such that the impacts of these various processes will affect the downstream and upstream concentrations equally. That is, in the upstream station, some amount of eDNA is coming from upstream of that location into the sampling station and leaving at the same time -- in the same way that eDNA would be both entering and exiting the downstream station. Additionally, at almost every single time point for all creeks and species, the upstream DNA concentration is higher than the downstream DNA concentration. Based on that alone, we do not expect that downstream accumulation of salmonid DNA is occurring to bias our results of whether fish can pass through these culverts.

### Not all culverts are barriers to salmonids

By measuring DNA concentrations of salmonid species above and below culverts on a small spatial scale, we were able to determine how much of a barrier each culvert was to fish passage. Barnes Creek was clearly a very large barrier to fish passage as we only found salmonid DNA in three months of the twelve months of sampling, and those three months had very low concentration of salmonid DNA relative to the other creeks. Within the treatment creek (Padden Creek), the SR-11 culvert did not seem to be a large barrier, while the I-5 culvert clearly was a barrier, demonstrated by the difference in salmonid composition and eDNA mass flow rates over the course of sampling.

Here, we find instances where culverts designated as barriers were likely not blocking fish passage, while others (Padden I-5 and Barnes Creek) were barriers to fish passage. Importantly, this demonstrates that collecting water samples for eDNA analysis might help to prioritize restoration of culverts suspected to be barriers to salmonids and provide a new method for post-restoration monitoring to confirm that the barrier has been corrected and allows for fish passage. Given the large amount of spending and effort required to replace culverts, this finding is important and emphasizes the potential for new tools for environmental impact assessments. We note that our sampling occurred only over a short temporal scale and future work could monitor culverts for longer time periods, different species, and different environmental conditions.

### Salmonids can quickly recover from a short-term intervention in a creek

The construction had remarkably minimal effects on salmonid DNA concentrations. The disruption of disconnecting the creek, demolition of the old culvert, installation of the new culvert, and the reconnecting of the creek during both culvert replacement events showed almost no change in the difference in eDNA concentrations between downstream and upstream sampling sites. The differences in the control creeks between upstream and downstream were often higher than the treatment creek. The post-construction sampling point of the I-5 culvert replacement (only one time point), does show that the composition of salmonid DNA after replacement is now very similar to the two downstream stations, whereas before construction compositions were very different (because the culvert was a barrier). However, we lack the quantitative analysis as the site upstream of SR-11 and downstream of I-5 had no quantifiable cutthroat DNA. More time points would help demonstrate the effect of the culvert replacement. Here we found that one culvert had very minimal effect on salmonid passage while the other culvert had a large effect on salmonid passage. We note that these findings are likely not universal and certainly projects need to monitor comprehensively and quantitatively in order to assess the passability of culverts and impacts of construction.

## Conclusion

It is notoriously difficult to quantify the environmental impact of discrete human impacts on ecosystems and species. Surveying species and communities by eDNA provides an opportunity for monitoring before, during, and after impacts in a scaleable and cost-effective way. Here, we demonstrate that monthly eDNA sampling before, during, and after an intervention alongside control sites can quantify the environmental impact of replacing a culvert. We found that in our treatment creek and control sites, four of the six barriers did not prohibit salmonid passage. We found that of the two culvert replacements in the treatment creek, one was a barrier and one was not, but both had minimal impacts on the four salmonid species monitored over the course of construction. We also provide a framework in which compositional metabarcoding data can be linked with qPCR data to obtain quantitative estimates of eDNA concentrations of many species. This provides a practical way to utilize the large amount of information from metabarcoding data without needing a unique qPCR assay for every species of interest. Environmental DNA is moving into practice and this study demonstrates how eDNA can be broadly used for environmental impact assessments for a wide range of species and environments.

## Conflict of Interest Statement

The authors declare there are no conflicts of interest.

## Acknowledgements

This work was made possible by a grant from OceanKind to Ryan Kelly. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript. Figure 3 was created with BioRender.com. We thank Tammy Schmidt and Susan Kanzler from Washington Department of Transportation for facilitating access to field sites and providing helpful feedback throughout the project. We also thank Jenna McLaughlin, Joe Duprey and Ally Im for help field sampling and Dr. Ramon Gallego, Dr. Kim Parsons, and the University of Washington's Northwest Genomics Center for sequencing support. Dr. Braeden Van Deynze, Dr. Sunny Jardine, and Dr. Julian Olden provided helpful insight into culverts and salmonid life histories. We thank Katherine Pearson Maslenikov at the Burke Museum of Natural History and Culture for providing voucher specimens for Sanger sequencing. Thanks to Dr. Jameal Samhouri and Dr. Chris Sergeant for reviewing the manuscript. 

\newpage

## Figures

\begin{itemize}
\setlength{\itemindent}{4em}
\item[\textbf{Figure 1.}] Map of sampling locations near Bellingham, Washington. In the red subset, triangles designate the downstream sampling location and circles designate the upstream sampling location. Padden Creek (the treatment creek where the two culverts were replaced) is shown in the blue subset where downstream is a triangle, upstream of the first culvert (SR-11) is a circle, and upstream of the second culvert (I-5) is a square.
\item[\textbf{Figure 2.}] Discharge (m\textsuperscript{3}/s) in Padden (USGS Gauge 12201905), Chuckanut (USGS Gauge 12201700), and Squalicum (USGS Gauge 12204010) Creeks over the course of sampling. Gauges at Chuckanut and Squalicum Creek went offline in November 2021 after a major storm event. Portage Creek and Barnes Creek did not have stream gauges. Circles designate the day of sampling. For Padden Creek, the nearest 15 minute interval of flow was used. For Chuckanut and Squalicum Creeks, the correction factor from five years of historical data from Padden Creek was used (see methods section and Appendix S1: Figure S2 and Appendix S1: Figure S3).
\item[\textbf{Figure 3.}] Conceptual figure of different datasets and models used for analyses. \* indicates that here, biological replicates are different dilutions of the synthetic gBlock. \*\* indicates that for most samples, only one technical replicate was sequenced but for one sample per sampling month, three technical replicates were sequenced to check for consistency across replicates. \*\*\* indicates that here, the three biological replicates indicate three different mock communities with varying species compositions, but all containing the four salmonids of interest. Created with BioRender.com.
\item[\textbf{Figure 4.}] Compositions of salmonid DNA in control creeks as determined by metabarcoding after correction for amplification bias. Grey shading denotes time points that were not sampled (Barnes and Chuckanut Creeks after March 2023 and Squalicum Creek in September 2021 which was dry). The empty bars in the Barnes upstream sites indicate that no salmonid DNA was found at those time points.
\item[\textbf{Figure 5.}] Compositions of salmonid DNA in Padden Creek as determined by metabarcoding after correction for amplification bias. The empty bar in the March 2023 in Up5 indicates that no salmonid DNA was found. The vertical dashed lines indicates the time period in which the culverts were replaced (SR-11 and I-5, respectively).]
\item[\textbf{Figure 6.}] Absolute mass flow rate (log copies/s) of cutthroat trout (\textbf{O. clarkii}) as measured by qPCR after flow correction. Note that Barnes Creek and Chuckanut Creek were not sampled after February 2022. Red crosses show the limit of detection for each species and time point, which changes with flow rate and total volume filtered per sample.
\item[\textbf{Figure 7.}] Trends in mass flow rate (log copies/s) for each of four salmonid species across creeks and across time as estimated by eDNA analysis. Points represent posterior means for the linear mixed effects model and error bars represent the 95% posterior confidence interval. Colors indicate station upstream (black) or downstream (grey) of the culvert. Padden has an additional sampling site upstream of the second culvert (I-5; blue). Yellow shading indicates the time period in which the culverts in the treatment creek (Padden Creek) were replaced. Grey shading indicates time points that were not sampled (Barnes and Chuckanut after February 2022). Time points with no data had no sequencing reads corresponding to that species or no quantifiable cutthroat DNA by qPCR.
\item[\textbf{Figure 8.}] The effect of culvert on salmonid abundance summed across all species and creeks by time. The y-axis shows the log-fold change in eDNA mass flow rate (copies/s) between upstream and downstream, normalized by upstream mass flow rate. The box boundaries correspond to the 25th and 75th percentiles; the whiskers correspond to 1.5 times the interquartile range. Here, negative values imply that eDNA mass flow rates are higher downstream than upstream. Samples with very low eDNA mass flow rates (less than 150 copies/s) were removed before plotting to remove extreme proportional values due to large denominators. Grey stars indicate times when no samples were taken.
\item[\textbf{Figure 9.}] Log-fold change in eDNA mass flow rate between upstream and downstream over time. Size of circles corresponds to the discharge in each creek at each time point. Color of circles corresponds to each creek. Each creek and time point has up to four circles of the same color for the four salmonid species.
\item[\textbf{Figure 10.}] Effect of construction on log-fold change in eDNA mass flow rate upstream to downstream in Padden Creek. Yellow shading shows when construction started for each of the two culverts. Grey points show the corresponding log-fold changes in control creeks and black points show Padden Creek. Sockeye/kokanee salmon (\textbf{O. nerka}) was only found in Padden Creek so other creeks are not shown. Samples with very low eDNA mass flow rates (less than 150 copies/s) were removed before plotting to remove extreme proportional values due to large denominators.

\end{itemize}

\newpage 
\textbf{Figure 1.}
![\label{fig:map}](../Output/Figures/SiteMap.png)
\newpage
\textbf{Figure 2.}
![\label{fig:flows}](../Output/Figures/flow_gauges.png)
\newpage
\textbf{Figure 3.}
![\label{fig:conceptualfig}](../Output/Figures/NGN_Fig3.png)
\newpage
\textbf{Figure 4.}
![\label{fig:qm_controls}](../Output/Figures/proportions_after_qm_nopadden.png)
\newpage
\textbf{Figure 5.}
![\label{fig:qm_padden}](../Output/Figures/proportions_after_qm_padden.png)
\newpage
\textbf{Figure 6.}
![\label{fig:qpcr}](../Output/Figures/modeled_cut_qpcr_updown_flowcorrected.png)
\newpage
\textbf{Figure 7.}
![\label{fig:ts}](../Output/Figures/multispeciesTrends_flowcorrected.png)
\newpage
\textbf{Figure 8.}
![\label{fig:culverts}](../Output/Figures/culvert_boxplot_passable.png)
\newpage
\textbf{Figure 9.}
![\label{fig:culverts_flow}](../Output/Figures/culvert_means_flow.png)
\newpage
\textbf{Figure 10.}
![\label{fig:construction}](../Output/Figures/culvert_padden_controls.png)
\newpage

## References
